docs/website/docs/user-guides/advanced/automounting-volumes.md

```diff
@@ -37,3 +37,4 @@ Mounting resources can be additionally configured via annotations:
 
 - `devfile.io/read-only`: for persistent volume claims, mount the resource as read-only
 
+- `devfile.io/mount-access-mode`: for  secret/configmap, can be used to configure file permissions on mounted files
```

Comment:
Can you also provide an example value? Along with the valid range?

This makes me wonder that we could add a complete example of an auto-mounting resource, with its labels and annotations. That would be helpful and easier to understand, IMO. But we can do that in a separate PR if you prefer..

Reply:
I've completed the doc, and added a test for decimal notation. 

I would prefer we discuss which kind of example we want and where to place it (blog, etc), and to create a separate PR for this.